src/runtime_src/core/include/ert.h

```diff
@@ -596,6 +596,13 @@ uint32_t ert_base_addr = 0;
 # define ERT_INTC_CU_96_127_ADDR           0x3000
 #endif
 
+	
+/**
+ * This defines ERT packet version which need to sync between Host and ERT 
+ */
+#define ERT_PACKET_VERSION_0		  0x51a10000
+#define ERT_PACKET_VERSION_1		  0x51a10001
```

Comment:
Thanks for adding the version check! I suggest to rename it to ERT_CUSTAT_VERSION other than the packet version.

Reply:
Updated. Thanks.